pkgs/applications/version-management/git-and-tools/git-team/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "git-team";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "hekmekk";
+    repo = "git-team";
+    rev = "v${version}";
+    sha256 = "0nl5j64b61jw4bkf29y51svjbndmqqrqx96yaip4vjzj2dx9ywm4";
+  };
+
+  vendorSha256 = null;
+
+  proxyVendor = true;
+
+  meta = with lib; {
+    description =
+      "Command line interface for managing and enhancing git commit messages with co-authors";
+    homepage = "https://github.com/hekmekk/git-team";
+    license = licenses.mit;
+    maintainers = with maintainers; [ lockejan ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
Default for `buildGoModule`.
```suggestion
```

Reply:
Ok, that works. 
I followed the example given in the docs: https://nixos.org/manual/nixpkgs/stable/#ssec-language-go

Feels like that that part should be mentioned in there too.